_includes/header.html

```diff
@@ -4,6 +4,17 @@
 
     {% assign custom_url = site.url | append: site.baseurl %}
     {% assign full_base_url = custom_url | default: site.github.url %}
+    {% if site.header_pages %}
```

Comment:
Good suggestion! Yes we could do that,
but i think we should use something like this then:
```
{% assign default_paths = site.pages | map: "path" %}
{% assign header_pages = site.header_pages | default: default_paths %}
```
To make sure in both cases we're dealing with `Strings` and not in one case `Strings`, in the other one with `Page` objects.

In the remaining loop we could then convert these back  to `Pages` one by one. What do you think?